checkout-core/src/test/java/com/adyen/checkout/core/model/JsonUtilsTest.kt

```diff
@@ -7,7 +7,7 @@
  */
 package com.adyen.checkout.core.model
```

Comment:
I guess this should be inside `com.adyen.checkout.core.internal.data.model` package